semantic/src/site/globals/site.overrides

```diff
@@ -1861,9 +1861,13 @@ a:hover {
     }
   }
 
-  // multiple is used to make sure filter menu stays open, but it adds extra margin on text
+  // multiple is used to make sure filter menu stays open, but it adds extra
+  // margin on text
   .ui.multiple.dropdown > .text {
-    margin: 0px;
+    margin: 0px ;    
+  }
+  .ui.multiple.dropdown {
+    padding: 5px 10px 5px 10px;
```

Comment:
As a result of the above change (id => class), the CSS precedence changes slightly (ids have higher priority than classes). So small adjustments required to keep it the same visually.